samples/bluetooth/mesh/light/README.rst

```diff
@@ -132,17 +132,36 @@ This sample is split into the following source files:
 DFU configuration
 =================
 
-To enable the DFU feature for the supported development kits, set :makevar:`SB_CONF_FILE` to :file:`sysbuild-dfu.conf` and set :makevar:`EXTRA_CONF_FILE` to :file:`overlay-dfu.conf` when building the sample.
-For example, when building from the command line, use the following command:
+.. tabs::
 
-  .. code-block:: console
+   .. tab:: nRF52840 DK and nRF54L15 PDK
 
-     west build -b <BOARD> -p -- -DSB_CONF_FILE="sysbuild-dfu.conf" -DEXTRA_CONF_FILE="overlay-dfu.conf"
+      To enable the DFU feature for the nRF52840 and nRF54L15 development kits, set :makevar:`SB_CONF_FILE` to :file:`sysbuild-dfu.conf` and :makevar:`EXTRA_CONF_FILE` to :file:`overlay-dfu.conf` when building the sample.
+      For example, when building from the command line, use the following command:
 
-The configuration overlay file :file:`overlay-dfu.conf` and the sysbuild configuration file :file:`sysbuild-dfu.conf` enable the DFU feature.
-To review the required configuration alterations, open and inspect the two files.
-For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
-For more information about selecting a sysbuild configuration file, see the Sysbuild Kconfig file section on the :ref:`zephyr:sysbuild` page in the Zephyr documentation.
+      .. code-block:: console
+
+         west build -b <BOARD> -p -- -DSB_CONF_FILE="sysbuild-dfu.conf" -DEXTRA_CONF_FILE="overlay-dfu.conf"
+
+      The configuration overlay file :file:`overlay-dfu.conf` and the sysbuild configuration file :file:`sysbuild-dfu.conf` enable the DFU feature.
+      To review the required configuration alterations, open and inspect the two files.
+      For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
+      For more information about selecting a sysbuild configuration file, see the Sysbuild Kconfig file section on the :ref:`zephyr:sysbuild` page in the Zephyr documentation.
```

Comment:
Is this information relevant for all DKs (both tabs)?
If yes, it can be then taken out of the tab and displayed after it :)